### DISCUSSION TITLE
大模型与 AIGC 的一些资料

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#306] [1️⃣ 新鲜事]
**1. 陆奇的报告**
- 新范式、新时代、新机会：https://www.youtube.com/watch?v=-LECKZqygzk
- [陆奇深圳演讲（2023年4月23日）.pdf](https://github.com/X-lab2017/open-wonderland/files/12470372/2023.4.23.pdf)

**2. 课代表立正**
- ChatGPT的理解、思考、与合理推演：https://www.youtube.com/watch?v=D1HRasV12mE
-  [关于ChatGPT的五个最重要问题.pdf](https://github.com/X-lab2017/open-wonderland/files/12470390/ChatGPT.pdf)

**Comment by bifenglin** [comment]
感觉语雀检索更容易。我创建了一个语雀知识库https://xlab2017.yuque.com/staff-kbz9wp/rf3sbn/er8f27gl9n3yc7ly
